core/schema/modulesource.go

```diff
@@ -1265,6 +1266,10 @@ func (s *moduleSchema) moduleSourceResolveDirectoryFromCaller(
 		}
 	}
 
+	if len(args.Ignore) > 0 {
+		excludes = append(excludes, args.Ignore...)
+	}
+
```

Comment:
I don't think this is a good idea. When specifying a view:
```
dagger call --source=.:default source
```
This code is adding the patterns from the view, with different semantics from `// +ignore` because of the split into exclude/include based on `!`, and adding the `// +ignore` patterns on top.

I think it's less surprising if you apply one or the other. Basically let a view override the patterns from code, instead of acting as a default.

Reply:
> This code is adding the patterns from the view, with different semantics from // +ignore because of the split into exclude/include based on !, and adding the // +ignore patterns on top.

And also because orders matter, it might totally break the expect result.

Reply:
Exactly.